resources/keb/migrations/202403051613_add_indexes.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX instances_by_created_at ON instances USING btree (created_at);
```

Comment:
Any. rationale why index for `instances` helps?
Could you attach at least explain without and with this index created?

Reply:
the `list` query has `order by instances.created_at` part